google-cloud-firestore/src/test/java/com/google/cloud/firestore/DocumentReferenceTest.java

```diff
@@ -255,6 +258,18 @@ public void deserializeBasicTypes() throws Exception {
     assertEquals(Timestamp.ofTimeSecondsAndNanos(5, 6), snapshot.getReadTime());
 
     assertEquals(get(), getAllCapture.getValue());
+    assertArrayEquals(SINGLE_FIELD_ARRAY, snapshot.getArray(Array_Value).toArray());
+    try {
+      snapshot.getArray(OBJECT_VALUE);
+    } catch (IllegalArgumentException exception) {
+      assertEquals("IllegalArgumentException", exception.getClass().getSimpleName());
```

Comment:
I think something like `assertTrue(exception.getMessage().contains("value"));` would be better.

Reply:
done.